modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -0,0 +1,80 @@
+
+If an API has only applied the Client ID Enforcement policy and not applied the OAuth policy the client application is still created in an external IdP.
+The API owner does not have access to the IdP configuration in API Manager. 
+
+When an OAuth policy is deployed to API gateway, the platform automatically injects IdP configuration, such as the token URL and token introspection url in the policy configuration. The IdP configuration is completely hidden from a user in API Manager.
+
+If no external IdPs are enabled, Access Manager acts as the default authorization server in that case. 
+//[PA:-Tomas not sure what the following statement is trying to say.]
+// Any client apps (clientID/clientSecret) that’re created for accessing an API are created and maintained within Anypoint.
+// OOTB OAuth policies with an external IdP can’t be applied 
+// ClientId enforcement policy can be applied.
+
+
+== The Use Case
+
+To better understand how to implement the multiple external credential provider feature, let us go back to our example of "Good Weather" company that is in the business of forecasting weather for a given location and time. Good Weather collects quantitative data about the current state of the atmosphere at a given place and, using meteorology, predicts how that current state will change over time.
+
+Good Weather has several sub-organizations running under the master organization. These sub-organizations exist based on the functions they perform, for example, finance, HR, and R&D. The R&D department researches and collects recent meterological data from various sources. This data is then consumed by various client applications. 
+
+Therefore, Good Weather needs to ensure that this data is accurate at all times. For that reason the weather APIs must be accessed only by employees of the R&D department. Good Weather also want to ensure that the production APIs and the single source of truth at all times.
+
+On the other hand, the HR sub-organization wants to ensure that the employees in R&D do not have access to their payroll APIs. HR also wants to restrict access to their internal tax and payroll APIs.
+
+To ensure security and segregation of access to these different organizations, environments, and APIs, Good Weather IT sets up a separate external IdP for each sub-organization and environments (production and non-production).
+
+
+== Guidelines for Implementing Multiple Client Providers
+
+Before you implement multiple client providers in your instance of Anypoint Platform, ensure that you carefully review the following guidelines and best practices:
+
+* If you assign an external IdP to an environment, all existing APIs in that environment display Anypoint Platform in the UI dropdown. 
```

Comment:
This lacks context and difficult to follow. Please clarify this statement

Reply:
Done! Thanks!